src/Symfony/Component/Validator/Tests/Mapping/Loader/AttributeLoaderTest.php

```diff
@@ -210,6 +210,22 @@ public function testLoadGroupSequenceProviderAttribute()
         $this->assertEquals($expected, $metadata);
     }
 
+    public function testLoadExternalGroupSequenceProvider()
+    {
+        $loader = $this->createAttributeLoader();
+        $namespace = $this->getFixtureAttributeNamespace();
+
+        $metadata = new ClassMetadata($namespace.'\GroupProviderDto');
+        $loader->loadClassMetadata($metadata);
+
+        $expected = new ClassMetadata($namespace.'\GroupProviderDto');
+        $expected->setGroupProvider('Symfony\Component\Validator\Tests\Dummy\DummyGroupProvider');
```

Comment:
Can we use `::class` here?

Reply:
It looks like the other tests use the same approach. Let’s improve them all in a separate PR. For now, let’s keep this consistent

Reply:
Fine for me